scrape/server/flask_app/repo/users.py

```diff
@@ -76,6 +92,45 @@ def delete_user_by_email(self):
         count = cursor.rowcount
         cursor.close()
 
+    # def validate_register(self, first_name: str, last_name: str, email: str, password: str):
+    #     is_valid = True
+    #     if not EMAIL_REGEX.match(email):
+    #         flash("Invalid Email Address")
+    #         is_valid = False
+    #     if len(first_name) < 3:
+    #         flash("First name must be at least 3 characters.")
+    #         is_valid = False
+    #     if len(last_name) < 3:
+    #         flash("Last name must be at least 3 characters.")
+    #         is_valid = False
+    #     if len(password) < 8:
+    #         flash("Password must be at least 8 characters.")
+    #         is_valid = False
+    #     return is_valid
+
+    def validate_register(user):
+        is_valid = True
+        user_in_db = User.get_user_by_email(user)
+        if user_in_db:
+            flash("Email is associated with another account")
```

Comment:
No need for `flash` here since we're separating frontend/backend and calling `flash` here won't do anything of importance. Instead, we can return an error for the frontend to flash